tools/stable-tag.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+BASE=$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+. "$BASE/tools/includes/check-osx-bash-version.sh"
+. "$BASE/tools/includes/chalk-lite.sh"
+. "$BASE/tools/includes/plugin-functions.sh"
+. "$BASE/tools/includes/proceed_p.sh"
+. "$BASE/tools/includes/version-compare.sh"
+
+# Instructions
+function usage {
+	cat <<-EOH
+		usage: $0 [options] <plugin>
+
+		Update the stable tag for the specified plugin. The <plugin> may be
+		either the name of a directory in projects/plugins/, or a path to a plugin directory or file.
+
+		Options:
+		  --dir <dir>  Use the specified directory for the SVN checkout,
+		               instead of creating a random directory in TMPDIR.
+
+	EOH
+	exit 1
+}
+
+# Process args.
+ARGS=()
+BUILD_DIR=
+while [[ $# -gt 0 ]]; do
+	arg="$1"
+	shift
+	case $arg in
+		--dir)
+			BUILD_DIR="$1"
+			shift
+			;;
+		--dir=*)
+			BUILD_DIR="${arg#--dir=}"
+			;;
+		--help)
+			usage
+			;;
+		*)
+			ARGS+=( "$arg" )
+			;;
+	esac
+done
+if [[ ${#ARGS[@]} -ne 1 ]]; then
+	usage
+fi
+
+$INTERACTIVE || die "Input is not a terminal, aborting."
+
+# Check plugin.
+process_plugin_arg "${ARGS[0]}"
+PLUGIN_NAME=$(jq --arg n "${ARGS[0]}" -r '.name // $n' "$PLUGIN_DIR/composer.json")
+WPSLUG=$(jq -r '.extra["wp-plugin-slug"] // ""' "$PLUGIN_DIR/composer.json")
+[[ -n "$WPSLUG" ]] || die "Plugin $PLUGIN_NAME has no WordPress.org plugin slug. Cannot deploy."
+
+# Get JSON
+JSON=$(curl -s "http://api.wordpress.org/plugins/info/1.0/$WPSLUG.json")
+if ! jq -e '.' <<<"$JSON" &>/dev/null; then
+	die "Failed to retrieve JSON data from http://api.wordpress.org/plugins/info/1.1/$WPSLUG.json"
+fi
+
+# Current stable version
+CURRENT_STABLE_VERSION=$(jq -r .version <<<"$JSON")
+
+# Get all versions, strip anything with alpha characters such as -beta or trunk.
+SVN_TAGS=$(jq -r '.versions | delpaths([paths | select(.[] | test("[A-Za-z]+"; "i"))]) | keys[]' <<<"$JSON")
+
+# Sort and create an array
+SVN_TAGS=( $(printf "%s\n" $SVN_TAGS | sort -V) )
+COUNT=${#SVN_TAGS[@]}
+SVN_LATEST=${SVN_TAGS[@]:$COUNT-1:1}
```

Comment:
I added this part because the JSON sorts versions alphabetically (1.0 10.0 11.0 2.0 3.0...). @anomiex If you think this is a good enough solution, we can add it to the revert-release.sh as well as it currently has a problem with double-digit versions.